init.lua

```diff
@@ -0,0 +1,3 @@
+require('lsp')
```

Comment:
Can you update readme to explain how to use and develop this extension?

Reply:
ok.

Reply:
I have updated readme. Please review again.